exporters/trace/src/main/java/com/google/cloud/opentelemetry/trace/TraceTranslator.java

```diff
@@ -28,9 +28,11 @@
 
   // TODO(nilebox): Extract the constant
   private static final String OPEN_TELEMETRY_LIBRARY_VERSION = "0.6.0";
+  private static final String EXPORTER_VERSION = "0.1.0";
   private static final String AGENT_LABEL_KEY = "g.co/agent";
   private static final String AGENT_LABEL_VALUE_STRING =
-      "opentelemetry-java [" + OPEN_TELEMETRY_LIBRARY_VERSION + "]";
+      "opentelemetry-java " + OPEN_TELEMETRY_LIBRARY_VERSION +
```

Comment:
The format of the agent attribute has been changed to `opentelemetry-<language_code> <ot_version>; google-cloud-trace-exporter <exporter_version>` (see [here](https://docs.google.com/document/d/1eeGeSYlE2F-Dc-5_yLXnIlj7e--BYFeWlzWy6qBV2cg/edit#heading=h.aoqug1blgfpl))